modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Actions.tsx

```diff
@@ -78,9 +79,25 @@ const Actions = () => {
 	] = useState({});
 	const [iconSymbol, setIconSymbol] = useState('bolt');
 
+	const request = new XMLHttpRequest();
+
+	request.open('GET', spritemap, false);
+
+	request.send();
```

Comment:
Our `fetch` utility is not covering this case is it?

Reply:
I don't think so, I'll try it out.

Reply:
It did work!